### PR TITLE
Move hard-coded configs to file.

### PR DESCRIPTION
**Pull request by dhananjay92**
Added Config Parser (similar to python scripts) for PHP. Now it reads all the configs from the same config file as python scripts do.

### CODE REVIEW

File: server/awfy-server.config [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [mysql]
-host = /var/run/mysqld/mysqld.sock
+host = localhost
 user = ???
 pass = ???
 name = ???
```
Comment (h4writer) [inline server/awfy-server.config]: While you are here. Can you rename to db_name?

File: website/internals.php [hunk 2]
```diff
@@ -5,10 +5,38 @@
...
+    }
+
+    function init($config_file_path)
+    {
+        // Read & Parse config file
+        $config_array = parse_ini_file($config_file_path, true);
+        $this->mysql_host     = $config_array["mysql"]["host"];
+        $this->mysql_username = $config_array["mysql"]["user"];
+        $this->mysql_password = $config_array["mysql"]["pass"];
+        $this->mysql_db_name  = $config_array["mysql"]["name"];
```
Comment (h4writer) [inline website/internals.php]: db_name

File: server/awfy-server.config (hunk 1 above)
Comment (h4writer) [inline server/awfy-server.config]: And also add:
data_folder = ???

**Comment by h4writer** [comment]
Yeah that will be much better. In that case I cannot accidentally push the password anymore ;).

**Comment by dhananjay92** [comment]
Yup. Renamed `name` to `db_name` and `path` to `data_folder`. Are there any other scripts that depend on these configs? Just let me know, I would change them as well.

**Comment by h4writer** [comment]
I think the only place that isn't changed is:
./detector/awfy.py

(which is a copy of ./server/awfy.py)

**Comment by dhananjay92** [comment]
Updated that one too. Please take a look.

**Comment by h4writer** [comment]
looks good!

**Comment by dhananjay92** [comment]
Then merge it please. This is pull to ypur master. :smile:

**Pull request merged**